### PR TITLE
Update NFT metadata oracle (CSV -> JSON)

### PR DESCRIPTION
**Pull request by GeorgeFlerovsky**
Changes:

- Apply [semantic line breaks](https://sembr.org/) to README, to improve the terminal-based editor experience and have better git diffs.
- Apply the assigned CIP number (86) to all our CIP documents.
- Replace CSV with JSON arrays in tabular updates.
- Allow tabular updates in `cddl/version_2.cddl`. (Now that we use JSON arrays, there's no problem in doing so)
- Add a section on versions 1 and 2 to README.
- Formatting fixes.

### CODE REVIEW

**Comment by GeorgeFlerovsky** [comment]
@nicolasayotte plz review and merge.

File: CIP-????/cddl/version_1.cddl [hunk 1]
```diff
@@ -1,8 +1,7 @@
-; "????" to be replaced by the CIP number, as a `uint`.
-metadata = { ???? => action }
+metadata = { "86" => action }
 
 ; =========================================================================
-; CIP-???? actions
+; CIP-86 actions
```
Comment (willbasky) [inline CIP-????/cddl/version_1.cddl:4]: Shouldn't it be `0086` as below
image

**Review by nicolasayotte — APPROVED** [review]

**Comment by nicolasayotte** [comment]
We changed the specs for the tabular updates (back) to JSON with a better formatting pattern and the cddl. This is not only much clearer to do and read, but also easier to parse AND results in smaller tx metadata.

File: CIP-????/cddl/version_1.cddl (hunk 1 above)
Comment (GeorgeFlerovsky) [reply CIP-????/cddl/version_1.cddl:4]: I'm not sure. People seem content to omit the leading zeroes when e.g. mentioning CIP-25...

**Pull request merged**